lib/hasher/include/SHA512Strategy.hpp

```diff
@@ -0,0 +1,26 @@
+#ifndef _SHA512_STRATEGY_HPP_
+#define _SHA512_STRATEGY_HPP_
+
+#include "HashStrategy.hpp"
+#include <string>
+#include <openssl/sha.h>
+
+namespace irods {
+    const std::string SHA512_NAME( "sha512" );
```

Comment:
This will result in multiple unique symbols appearing in the shared libs. This line should be replaced with a declaration:
```c++
extern const std::string SHA512_NAME;
```
The definition should appear in the cpp file like so:
```c++
const std::string SHA512_NAME( "sha512" )
```
These changes need to be applied for all of the hashing hpp and cpp files.

Reply:
Done for all hashing schemes.